cg/cli/workflow/balsamic/base.py

```diff
@@ -197,28 +197,26 @@ def start(
     dry_run: bool,
 ):
     """Start full workflow for case ID."""
+    analysis_api: BalsamicAnalysisAPI = context.obj.meta_apis["analysis_api"]
+    analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
     LOG.info(f"Starting analysis for {case_id}")
-    try:
```

Comment:
Can this `try` be skipped now?

Reply:
We decided on letting the start command raise an error and to let start-available catch that error. Would it not feel weird if you could fail to start an analysis and no error was raised?